### ISSUE TITLE
How to use a Weak-Perspective projection

### ISSUE THREAD

**Issue by adrianofragomeni** [daniilidis-group/neural_renderer#77]
Hello,
I need to render a mano mesh into its silhouette, where the predicted mesh is already aligned with the camera view, but it is root centred. Moreover, I do not have information about the intrinsic camera parameters, therefore I decided to use a weak-perspective projection. I am wondering if it is possible to modify the projection file as follows: Given `scale` as the scale factor and `trans` as a 2D translation vector and `orig_size` as the original size of image captured by the camera

    proj = scale.unsqueeze(1) * vertices[:, :, :2]

    proj_xy = proj + 100*trans.unsqueeze(1)
    proj_z = vertices[:, :, 2]
    u, v = proj_xy[:, :, 0], proj_xy[:, :, 1]

    v = orig_size - v
    # map u,v from [0, img_size] to [-1, 1] to use by the renderer

    u = 2 * (u - orig_size / 2.) / orig_size
    v = 2 * (v - orig_size / 2.) / orig_size

    vertices = torch.stack([u, v, proj_z], dim=-1)

    return vertices

Thank you

**Comment by YfDebug** [comment]
@adrianofragomeni Have you solved this problem? I have the same problem！
